### ISSUE TITLE
Unable to access rancher UI.

### ISSUE THREAD

**Issue by rajboruah** [yamamoto-febc/terraform-provider-rke#19]
Hi,

I used the new provider and was able to get the k8s cluster up & running with rancher.
But wondering, how do I modify the ingress rules to access the rancher UI ?
I wanted to use something like below to access the rancher UI:

host: mykubernetes.com

I also do not see any k8s service running for rancher UI.

I used the master/examples/full/example.tf and modified it for my use and installed the cluster in single VM.

**Comment by yamamoto-febc** [comment]
According to your tf file, it seems that  Rancher UI is not deployed.

If you want to use Rancher UI, please use following examples:

 - [Use own SSL certificates](https://github.com/yamamoto-febc/terraform-provider-rke/tree/0.3.4/examples/rancher_server_minimal/example.tf)
 - [Use SSL-passthrough](https://github.com/yamamoto-febc/terraform-provider-rke/tree/0.3.4/examples/rancher_server_ssl_passthrough/example.tf)

**Comment by rajboruah** [comment]
Thanks. I added rancher-ui part into master/examples/full/example.tf  and it worked fine. I am able to access rancher-dashboard using ingress now.

But as I saw in previous rancher too, kube-dns keeps crashing. Had to apply the below config map (this seems to be an issue sky-dns):

apiVersion: v1
kind: ConfigMap
metadata:
  name: kube-dns
  namespace: kube-system
  labels:
    addonmanager.kubernetes.io/mode: EnsureExists
data:
  upstreamNameservers: |-
    ["8.8.8.8", "8.8.4.4"]
